apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt_connector_schema.erl

```diff
@@ -370,6 +377,45 @@ desc(static_clientid_entry) ->
 desc(_) ->
     undefined.
 
+injected_fields() ->
+    #{
+        'connectors.validators' => [fun ?MODULE:unique_static_clientid_validator/1]
+    }.
+
+unique_static_clientid_validator(#{<<"mqtt">> := MQTTConnectors}) ->
```

Comment:
should take target `host` into consideration ?
the same clientid(s) should be allowed when connecting to different brokers.